addons/api/addon/models/role.js

```diff
@@ -27,7 +27,7 @@ export default class RoleModel extends GeneratedRoleModel {
    * see obvious.  Instead, the application layer is expected to load referenced
    * users and groups as needed.
    */
-  @attr('principal-array', {
```

Comment:
I refactored this to a generic array which can be used in other resources as well.